go.mod

```diff
@@ -43,6 +44,13 @@ require (
 	sigs.k8s.io/controller-runtime v0.16.0
 )
 
+require (
+	github.com/zhengxiexie/vsphere-automation-sdk-go/lib v0.7.2
```

Comment:
do we merge this change now, or wait till new SDK ready and change it to official one?